platform/linux/src/headless_backend_glx.cpp

```diff
@@ -39,7 +39,7 @@ class GLXDisplayConfig {
             if (xDisplay != nullptr) {
                 break;
             }
-            Log::Debug(Event::OpenGL, "[CI] Failed to open X display, retrying...");
+            Log::Error(Event::OpenGL, "[CI] Failed to open X display, retrying...");
```

Comment:
This was changed from Error to Debug intentionally, some of the tests use a [log observer](https://github.com/maplibre/maplibre-native/blob/88917b18065f6c7ee5d11ab2e109e0fab7af6edf/src/mbgl/util/logging.cpp#L53-L56) and only [Debug is omitted](https://github.com/maplibre/maplibre-native/blob/88917b18065f6c7ee5d11ab2e109e0fab7af6edf/src/mbgl/util/logging.cpp#L79). If XOpenDisplay fails during a test using the [FixtureLogObserver](https://github.com/maplibre/maplibre-native/blob/main/test/src/mbgl/test/fixture_log_observer.hpp), the test will fail because of the error level used in this retry message.

Reply:
Thanks for clarifying.

I didn't see any retry messages in the logs, so I didn't know if it was actually retrying.